package.json

```diff
@@ -19,9 +19,8 @@
   },
   "license": "MPL-2.0",
   "dependencies": {
-    "@cliqz/indexeddbshim": "3.4.1",
     "appium": "1.9.0",
-    "browser-core": "http://cdn2.cliqz.com/update/edge/ghostery-ios/v3.31/3.31.2.2e55387.tgz",
+    "browser-core": "https://s3.amazonaws.com/cdncliqz/update/edge/ghostery-ios/master/latest.tgz",
```

Comment:
I think it is better to use specific version not latest version which is changing periodically

Reply:
Yep that's exactly why this PR is WIP. We are waiting for extension to be stabilized, so that we can branch out and point to a specific version.

Reply:
Oh I didn't notice `[WIP]` tag

Reply:
Sure :-) I have many dependencies for this PR a) navigation extension stable branch, b) indexeddbshim node package to be pushed to npm registry by @chrmod